### PR TITLE
fix: add proper types

### PR DESCRIPTION
**Pull request by rhlsthrm**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

**Review by prathmeshkhandelwal1 — APPROVED** [review]
Looks good to me

File: packages/agents/sdk/src/lib/helpers/shared.ts [hunk 1]
```diff
@@ -55,3 +55,59 @@ export const parseConnextLog = (log: providers.Log): any => {
...
+  origin_sender: string;
+  bridged_amt: string;
+  normalized_in: string;
+  canonical_id: string;
+  router_fee?: string;
+  xcall_tx_origin: string;
+  execute_tx_origin: string;
+  reconcile_tx_origin: string;
+  relayer_fee: string;
+  error_status?: string;
```
Comment (sanchaymittal) [inline packages/agents/sdk/src/lib/helpers/shared.ts:112]: execute_simulation_input?: string,
    execute_simulation_from?: string,
    execute_simulation_to?: string,
    execute_simulation_network?: string,
    error_message?: string,
    message_status?: string,
    relayer_fees?: string[]

Comment (sanchaymittal) [reply packages/agents/sdk/src/lib/helpers/shared.ts:112]: added the ones i think was missing, @coinhippo review is required here as well.

**Review by sanchaymittal — CHANGES_REQUESTED** [review]

File: packages/agents/sdk/src/lib/helpers/shared.ts (hunk 1 above)
Comment (sanchaymittal) [reply packages/agents/sdk/src/lib/helpers/shared.ts:112]: @nrsirapop @pjenviri Can we check if all the types are present which we are using or need.

Comment (sanchaymittal) [reply packages/agents/sdk/src/lib/helpers/shared.ts:112]: for list ref: https://github.com/connext/monorepo/blob/0ebcfe1fdf6a2687eecf7de304c46df7ea4b0b89/packages/adapters/database/db/schema.sql

Comment (rhlsthrm) [reply packages/agents/sdk/src/lib/helpers/shared.ts:112]: Added the missing ones.

**Review by sanchaymittal — APPROVED** [review]

**Pull request merged**